homeassistant/components/ollama/const.py

```diff
@@ -81,6 +81,11 @@
 CONF_MAX_HISTORY = "max_history"
 DEFAULT_MAX_HISTORY = 20
 
+CONF_NUM_CTX = "num_ctx"
+DEFAULT_NUM_CTX = 2048
+MAX_NUM_CTX = 65536
```

Comment:
Newer model seem to support up to 128K, might be a good idea to change max to an even higher number (unless I'm misunderstanding something).

Reply:
Perhaps.  Mind suggesting a diff / patch in this PR?  I would happily increase it.

Reply:
```suggestion
MAX_NUM_CTX = 131072
```